.eslintrc.json

```diff
@@ -82,7 +83,10 @@
           { "object": "process", "property": "exitCode" }
         ]
       }
-    ]
+    ],
+    "ava/no-ignored-test-files": [2, { "files": ["tests/**/*.js", "!tests/{helpers,fixtures}/**/*.{js,json}"] }],
+    "ava/no-import-test-files": [2, { "files": ["tests/**/*.js", "!tests/{helpers,fixtures}/**/*.{js,json}"] }],
```

Comment:
Ava default values to distinguish test files from test helpers and fixtures does not match ours, so we need to configure it here.